core/src/test/scala/integration/kafka/api/ProducerFailureHandlingTest.scala

```diff
@@ -188,7 +188,7 @@ class ProducerFailureHandlingTest extends KafkaServerTestHarness {
   @Test
   def testInvalidPartition() {
     // create topic with a single partition
-    TestUtils.createTopic(zkClient, topic1, 1, numServers, servers)
+    createTopic(topic1, replicationFactor = numServers)
```

Comment:
nit: since this test case depends on the topic having one partition, maybe we can pass that explicitly to `createTopic` instead of depending on the default.